mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -2346,41 +2346,29 @@ private void MakeAttributes (XmlElement root, IEnumerable<string> attributes, Ty
 	}
 
 	public static string MakeAttributesValueString (object v, TypeReference valueType)
-	{
-		if (v == null)
-			return "null";
-			if (valueType.FullName == "System.Type") {
-				var vTypeRef = v as TypeReference;
-				if (vTypeRef != null)
-					return "typeof(" + NativeTypeManager.GetTranslatedName (vTypeRef) + ")"; // TODO: drop NS handling
-				else
-					return "typeof(" + v.ToString () + ")";
+		{
```

Comment:
ack! man, yeah Xamarin Studio makes it so hard to adhere to the existing formatting in this file ... I will submit another change that fixes this, but will wait until you're done looking at the rest of the changes :)